### PR TITLE
ksonnet: the command has moved into cmd/ks as of 0.10.0

### PR DESCRIPTION
**Pull request by kalbasit**

### CODE REVIEW

**Review by adamdecaf — APPROVED** [review]

**Comment by adamdecaf** [comment]
cc @nikolay thoughts?

**Comment by adamdecaf** [comment]
@kalbasit There looks to be a problem with the CI, could you take a look?

```
/tmp/asdf.ZDRo/plugins/ksonnet/bin/install: line 36: master: unbound variable
FAILED: install exited with an error
```

File: bin/install [hunk 1]
```diff
@@ -33,14 +33,20 @@ install () {
     local -r source_base_path="${TMP_DOWNLOAD_DIR}/src/github.com/ksonnet"
     local -r source_path="${source_base_path}/${toolname}"
 
+    if [[ "$(echo $version | cut -d. -f2)" -ge 10 ]]; then
+      local -r bindir="${source_path}/cmd/ks"
```
Comment (DustinChaloupka) [inline bin/install]: Does `local` set this to the scope of the `if/else` statement, or to the function?

Comment (kalbasit) [reply bin/install]: the problem was in the `$(echo $version | cut -d. -f2)` which is creating a variable inside the subshell `[[ ]]`. I've fixed this now and will squash my commits and update my PR.

**Comment by kalbasit** [comment]
@adamdecaf the CI is now green.

**Comment by nikolay** [comment]
Thank you, @kalbasit! To be consistent though, is it possible to rename `bindir` to `bin_path`?

**Comment by kalbasit** [comment]
@nikolay it's done.

**Comment by adamdecaf** [comment]
Awesome, thanks!

**Pull request merged**